### PR TITLE
[X86] Fix incorrect weight scale in fp8 qconv

### PR DESCRIPTION
**Pull request by Yuxingwang-intel**
Because the same buffer name was reused, the later-registered convolution overwrote the previous one, causing multiple convolutions to incorrectly share the same weight scale, which led to accuracy issues in FP8 qconv.

### CODE REVIEW

**Comment by Yuxingwang-intel** [comment]
@Xia-Weiwen Could you please help me review this PR. Thanks!

File: torchao/quantization/pt2e/inductor_passes/x86.py [hunk 1]
```diff
@@ -795,8 +795,9 @@ def qconv_weight_prepack(match: Match, *args, **kwargs):
             )
             with torch.utils._python_dispatch._disable_current_modes():
                 w_scale_tensor = torch.tensor([w_scale.args[1]])
-            match.graph.owning_module.register_buffer("w_scale", w_scale_tensor)
-            w_scale = match.graph.create_node("get_attr", "w_scale")
+            scale_name = f"{conv_node}"
```
Comment (Xia-Weiwen) [inline torchao/quantization/pt2e/inductor_passes/x86.py]:
```suggestion
            scale_name = f"{conv_node}_scale"
```
seems better?

**Review by Xia-Weiwen — APPROVED** [review]

**Comment by Xia-Weiwen** [comment]
Can you add a test case for it?

**Comment by Xia-Weiwen** [comment]
The CI failure is irrelevant to this PR. And we plan to enable correctness test for quantization path in another PR since it is clearer.
